odoo/addons/base/tests/test_translate.py

```diff
@@ -376,6 +376,25 @@ def setUpClass(cls):
             translation_importer.load(f, 'po', 'fr_FR')
             translation_importer.save(overwrite=True)
 
+    def test_101_translation_read(self):
```

Comment:
Maybe you can try some SQL injection attempt, in order to check that the `SQL` wrapping does its job (for reading at least).
```py
# if it succeeds, this attempt of an SQL injection would make the fetching of data fail because of the missing parenthesis
category.with_context(lang="'', NOW(').name
```